static/js/entry/public_jquery.js

```diff
@@ -0,0 +1,6 @@
+__webpack_public_path__ = `${SETTINGS.public_path}` // eslint-disable-line no-undef, camelcase
```

Comment:
`public.js` requires these lines to be executed and previously they were executed through `style_public.js`. `style_public` and link sharing `rrssb` buttons depend upon `is_public` context passed from view to template but in our scenario as the record page `/records/programs/enrollment_id/` is not going to be shared with any one so we can't pass `is_public` which in return causes a crash of `public.js`. So this file was included just to fulfill that requirement.